raiden/tests/unit/api/test_python_api.py

```diff
@@ -156,3 +160,55 @@ def test_api_channel_events(raiden_chain):
         channel_0_1.channel_address, max_block + 1, max_block + 100
     )
     assert not results
+
+
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('register_tokens', [True, False])
+@pytest.mark.parametrize('settle_timeout', [600])
+def test_channel_open_close(raiden_network, token_addresses, settle_timeout):
+    app0, app1 = raiden_network
+
+    maker_address = app0.raiden.address
+    taker_address = app1.raiden.address
+
+    token_address = token_addresses[0]
+
+    api0 = RaidenAPI(app0.raiden)
+    api1 = RaidenAPI(app1.raiden)
+    api0.open(
+        token_address,
+        taker_address,
+        settle_timeout,
+    )
+
+    # wait for the taker to receive and process the messages
+    gevent.sleep(1)
+
+    channel0 = channel(app0, app1, token_address)
+    channel1 = channel(app1, app0, token_address)
+
+    assert channel0 in api0.get_channel_list(
+        partner_address=taker_address,
+        token_address=token_address,
+    )
+    assert channel1 in api1.get_channel_list(
+        partner_address=maker_address,
+        token_address=token_address,
+    )
+
+    api0.close(token_address, taker_address)
+
+    gevent.sleep(1)
+
+    assert all(channel.state in (CHANNEL_STATE_CLOSED, CHANNEL_STATE_SETTLED)
```

Comment:
It was @LefterisJP suggestion, since although I don't call settle, `CHANNEL_STATE_SETTLED` is a following state of `CHANNEL_STATE_CLOSED`, so even if the test is a little bit loose, it doesn't get any edge case on it.